### ISSUE TITLE
v0.8+ is broken

### ISSUE THREAD

**Issue by andrewl913** [vitest-dev/vscode#357]
### Describe the bug

v0.8 of the extension is broken.

v0.6 is latest working version

### Reproduction

Install any version  v0.8 and above.

Running on MacOS Montery 

### Output

```shell
: Vitest failed to start: 

r
	at ChildProcess.u (/Users/[redacted]/.vscode/extensions/vitest.explorer-0.8.0/dist/extension.js:13:331)
	at ChildProcess.emit (node:events:517:28)
	at emit (node:internal/child_process:944:14)
	at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
``
```

### Version

0.8

### Validations

- [X] Check that you are using the latest version of the extension
- [X] Check that there isn't [already an issue](https://github.com/vitest-dev/vscode/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vitest-dev/vscode/discussions) or join our [Discord Chat Server](https://chat.vitest.dev).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by sheremet-va** [comment]
> TL; DR - I had node v16.x.x installed (globally), updated to v20.x.x and it was fixed.

Yes, the extension now uses the local Node.js executable resolved relative to the workspace root using `which` command:

https://github.com/vitest-dev/vscode/blob/6a103f7fd395c53575af01c49e37be3dd21fd446/src/utils.ts#L93

Make sure your Node.js version is matching minimal Vitest requirements (Node >=v18.0.0)

**Comment by guysympoly** [comment]
> Yes, the extension now uses the local Node.js executable resolved relative to the workspace root using `which` command:

I did actually look into it. It has to be git-bash (windows) vs CMD. running `node -v` on git-bash points to the version installed by fnm, and running in CMD points to the one in Program Files.

**Comment by theClarkSell** [comment]
FWIW I ran into this as well. macos using nodenv. We were already running 18 in our project. I'm not sure what version it picked up, but once I set a global 0.8 worked. Something I need to look into more.
